MyLibrary/Sources/ScheduleFeature/Schedule.swift

```diff
@@ -93,6 +96,14 @@ public struct Schedule {
         #elseif os(visionOS)
           return .run { _ in await openURL(url) }
         #endif
+      case let .fetchResponse(.success((day1, day2, workshop))):
+        state.day1 = day1
+        state.day2 = day2
+        state.workshop = workshop
+        return .none
+      case let .fetchResponse(.failure(error)):
+        print(error)
+        return .none
```

Comment:
Now that we are pulling data from local JSON, we would like to notice at DEBUG mode if any data is corrupt.

DecodingError
```suggestion
      case let .fetchResponse(.failure(error as DecodingError)):
         assertionFailure(error.localizedDescription)
         return .none
      case let .fetchResponse(.failure(error)):
        print(error) // TODO: replace to Logger API
        return .none
```